lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -323,7 +331,7 @@ int Ohmbrewer::Thermostat::doDisplay(Ohmbrewer::Screen *screen) {
     // Print the section title
     screen->print("== Thermostat #");
     screen->print(getID());
-    screen->println(" ==");
+    screen->print("  ==");
```

Comment:
It works for me, though now that you mention it, that does seem strange. 
The Screen::printMargin method does call println, so I'm wondering if the "right" way to do that would be to do a println here and remove the printMargin on line 337.